woof-code/rootfs-skeleton/usr/sbin/snapmergepuppy

```diff
@@ -204,12 +204,18 @@ do
 		if [ -L "$N" ] && [ -d "$BASE/$N" ] ; then
 			rm -rf "$BASE/$N"	# SFR: in case if folder has been replaced with a symlink (cp won't overwrite a dir with a symlink)
 		fi
-		cp -af "$N" "$BASE/$N" 2>>/tmp/snapmergepuppy-error
-		rm "$N"
+		#Move if the file was not opened by a process
+		if [ "$(lsof | awk '{ print $3 }' | grep "/" | grep -vE "\/dev|\/proc|\/sys|\/tmp|\/run|\/mnt|\/media" | sort | uniq | grep "$N")" == "" ]; then
+		 cp -af "$N" "$BASE/$N" 2>>/tmp/snapmergepuppy-error
+		 rm "$N"
```

Comment:
Maybe just copy but without deleting, if the file is open? That's safer against power failure. For example, if the browser is constantly open, the user will lose all browsing history since bootup when the battery drops to zero. On the other hand, there's a race condition between the process that writes to the file and `cp`, so the result of the copying is not guaranteed to be a valid file.

Reply:
> Maybe just copy but without deleting, if the file is open?

Not a good idea for me.